source/functionality/api_reference.md

```diff
@@ -0,0 +1,35 @@
+# Enable API reference 
+
+This feature extracts endpoint information from an Open API v3 yaml file that describes an API.
+
+## Amend the tech-docs.yml file
+
+Add the following to your tech-docs.yml file:
+
+```
+api_path: path/to/file.yaml
+```
+
+This can be a relative path to a file in your tech docs repo folder, for example:
```

Comment:
Presumably it has to be an absolute _or_ a relative path if it's not specified which: this is a little ambiguous; I'm not sure it's necessary given the above guidance about `api_path: path/to/file.yaml`

Reply:
hmm. I think it's ok but don't feel strongly about it

Reply:
I think it has to be a relative path, from memory? I haven't looked in a while so not sure. If it can be both then fair enough!